docker-compose.yml

```diff
@@ -17,6 +17,9 @@ sales:
     image: autopilotpattern/workshop-sales
     links:
       - consul
+    environment:
+      - CONSUL_HOST=consul
```

Comment:
Good idea. The next PR will be to more closely integrate with triton CNS and remove the links.